### DISCUSSION TITLE
TouchableOpacity with borderRadius on hover inside the simulator, the hover overal does not show borderRadius

### DISCUSSION THREAD

**Discussion by bartlomein** [callstack/react-native-visionos#116] [General]
I don't have a visionPro device so I am not sure if anyone else experience that, and if its just a simulator issue. Anyone else can see it?

**Comment by okwasniewski** [comment]
Hey, can you try out the new architecture (installing cocoapods with `RCT_NEW_ARCH_ENABLED=1 bundle exec pod install`). I think I saw this issue on the old architecture but I didn't find any solution yet

**Reply by bartlomein** [reply]
I tried that and now building the app after running it throws errors:

** BUILD FAILED **

The following build commands failed:
	CompileC /Users/bart/Library/Developer/Xcode/DerivedData/mmvisionpro-dvmcvsdemvrlkedlvtuvxhvkzrst/Build/Intermediates.noindex/Pods.build/Debug-xrsimulator/React-RCTAppDelegate.build/Objects-normal/arm64/React-RCTAppDelegate-dummy.o /Users/bart/dev/mmvisionpro/visionos/Pods/Target\ Support\ Files/React-RCTAppDelegate/React-RCTAppDelegate-dummy.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'React-RCTAppDelegate' from project 'Pods')
(1 failure)

**Reply by okwasniewski** [reply]
Can you cleanup the build folder using `git clean -xdf visionos` and reinstall pods? Also, can you try building from Xcode is it provides more descriptive error messages

**Reply by bartlomein** [reply]
That worked! Thanks so much @okwasniewski ! The original problem is gone, borderRadius shows correctly.
